README.md

```diff
@@ -83,16 +83,16 @@ questions when you write a review:
 ## 🎤 Stage
 
 Have you recently been a speaker at a local meetup or at a conference?
-Open the [Stage](https://github.com/hyperoslo/hyper-radar/blob/master/Stage)
-folder and share your keynote, so everyone can enjoy the slides and re-visit it
-on any day, at any time.
+Open the [Stage](https://github.com/hyperoslo/hyper-radar/blob/master/Library/stage.md)
+document and share a link to your keynote, so everyone can enjoy the slides
+and re-visit it on any day, at any time.
 
 ## 🦄 Inspiration
 
-Use (https://github.com/hyperoslo/hyper-radar/blob/master/inspiration.md) to
-share links to all the interesting reads you've found on the internet: popular
-tech-related stories, articles and everything that can inspire and push us
-forward.
+Use (https://github.com/hyperoslo/hyper-radar/blob/master/Library/inspiration.md)
```

Comment:
Did you forget the link text for this one? Or did you mean to use `<` and `>` instead?